_includes/head.html

```diff
@@ -70,9 +70,18 @@
   <script defer src="/assets/js/docs.js"></script>
   <script defer src="/assets/js/copy.js"></script>
   {%- endif -%}
-  {%- if site.local_search -%}
-  <script defer src="/assets/js/search.js"></script>
-  {%- endif -%}
+  <script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
+  <script defer type="text/javascript">
+  window.addEventListener('DOMContentLoaded', function() {
+    docsearch({
+      appId: "3XRLW0LZH9",
```

Comment:
We should probably consider putting these IDs in the _config.yml file (can the same ID be used for both staging/previews and production? Or should search be disabled on the previews?

Reply:
Good catch, yes I agree – should definitely go in the site config.

The search results only link to docs.docker.com - our index gets populated by a crawler, so we won't have any 'local' search that works with staging/previews. But I still think we can have the production search enabled for all builds regardless.

Reply:
Yes, we can have a look where to enable it; one of the reason we disabled some of the features was also because we encountered various users either forking the docs (for other websites), or hosting a copy of the docs somewhere; such websites would also be connected to our IDs, which gave some fun results 😅